packages/showcase/src/app/components/drop-down/metadata.ts

```diff
@@ -0,0 +1,399 @@
+import { ComponentMetadata } from "../../../common/types";
+import { chameleonImportType } from "../../../services/template-language/create-template";
+import {
+  nestedModel,
+  plainModel,
+  withSeparatorsModel,
+  customContentModelExampleGenericMetadata,
+  withIconsModelForMetadata,
+  withLinksModel
+} from "./models";
+import { CSS } from "../../../common/html";
+
+// Common
+const CHILDREN = "Click here";
+const PROPERTIES = [
+  { name: "model", value: "actionMenuModel", state: true },
+  { name: "inlineAlign", value: "inside-start" },
+  { name: "blockAlign", value: "outside-end" }
+];
+const IMPORTS_NO_ICONS = [chameleonImportType("ActionMenuModel")];
+
+export const dropDownMetadata = {
+  title: "Drop Down",
+  description:
+    "The dropdown component is an interactive element used to display a list of options from which users can select. It supports various states (open, disabled, focused) and styles (standard, outlined, filled) for customization and accessibility.",
+  codeSnippets: {
+    plain: {
+      linkId: "plain",
+      title: "1. Plain",
+      imports: IMPORTS_NO_ICONS,
+      states: [
+        {
+          name: "actionMenuModel",
+          type: "ActionMenuModel",
+          value: plainModel
+        }
+      ],
+      template: {
+        tag: CSS.DROPDOWN.TAG,
+        class: CSS.DROPDOWN.SELECTOR,
+        properties: PROPERTIES,
+        children: CHILDREN
+      }
+    },
+
+    nested: {
+      linkId: "nested",
+      title: "2. Nested",
+      imports: IMPORTS_NO_ICONS,
+      states: [
+        {
+          name: "actionMenuModel",
+          type: "ActionMenuModel",
+          value: nestedModel
+        }
+      ],
+      template: {
+        tag: CSS.DROPDOWN.TAG,
+        class: CSS.DROPDOWN.SELECTOR,
+        properties: PROPERTIES,
+        children: CHILDREN
+      }
+    },
+
+    withSeparators: {
+      linkId: "with-separators",
+      title: "3. With Separators",
+      imports: IMPORTS_NO_ICONS,
+      states: [
+        {
+          name: "actionMenuModel",
+          type: "ActionMenuModel",
+          value: withSeparatorsModel
+        }
+      ],
+      template: {
+        tag: CSS.DROPDOWN.TAG,
+        class: CSS.DROPDOWN.SELECTOR,
+        properties: PROPERTIES,
+        children: CHILDREN
+      }
+    },
+
+    customContentExample1: {
+      linkId: "custom-content-example-1",
+      title: "4. Custom Content: Example 1",
+      imports: IMPORTS_NO_ICONS,
+      states: [
+        {
+          name: "actionMenuModel",
+          type: "ActionMenuModel",
+          value: customContentModelExampleGenericMetadata
+        }
+      ],
+      template: {
+        tag: CSS.DROPDOWN.TAG,
+        class: CSS.DROPDOWN.SELECTOR,
+        properties: PROPERTIES,
+        children: [
+          "Click me",
+          {
+            tag: "div",
+            properties: [{ name: "slot", value: "my-custom-content" }],
+            children: "[You custom content here]"
```

Comment:
Please, add the explicit code here to render the preview you are showing in the showcase.